book/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+import json
+from datetime         import timedelta, date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Book
+
+
+class RecentlyBookView(View):
+    def get (self, request):
+
+        today          = date.today()
+        previous_days  = today - timedelta(days=30)
```

Comment:
```suggestion
        previous_days  = today - timedelta(days=day)
```
이 엔드포인트를 필요할 때에 '일년 이내 출간된 책', '3개월 이내 출간된 책' 등으로 확장해서 사용할 수 있도록 날짜의 수를 30으로 하드코딩 해두시기 보다는 프론트엔드로부터 날짜 수를 변수로 받아오시는 방법을 추천드려봅니다 :)

Reply:
쿼리스트링 처리로 수정했습니다. 감사합니다.